docs/topics/native/native-objc-interop.md

```diff
@@ -16,6 +16,25 @@ Kotlin modules can be used in Swift/Objective-C code if compiled into a
 framework ([see here for how to declare binaries](multiplatform-build-native-binaries.md#declare-binaries)).
 See [Kotlin Multiplatform Mobile Sample](https://github.com/Kotlin/kmm-basic-sample) for an example.
 
+### Hiding Kotlin declarations
+
+If you don't want to export Kotlin declarations to Objective-C and Swift, use special annotations:
+
+* `@HiddenFromObjC` hides a Kotlin declaration from Objective-C and Swift. The annotation disables a function or property
+  export to Objective-C, making your Kotlin code more Objective-C/Swift-friendly.
+* `@ShouldRefineInSwift` helps to replace a Kotlin declaration with a wrapper written in Swift. The annotation marks a
+  function or property as `swift_private` in the generated Objective-C API. Such declarations get the `__` prefix,
+  which makes them invisible from Swift.
+
+  You can still use these declarations in your Swift code to create a Swift-friendly API, but they won't be suggested in
```

Comment:
@danil-pavlov is it deliberate that these two paragraphs are in the second list item?

https://kotlinlang.org/docs/native-objc-interop.html#hiding-kotlin-declarations

<img width="782" alt="Screenshot 2022-12-28 at 19 24 12" src="https://user-images.githubusercontent.com/78360457/209856265-9e1d0425-8846-471c-a755-61141d15bbc1.png">

Reply:
Yes, it is deliberate. thank you!